packages/webpack/src/plugin.js

```diff
@@ -548,6 +588,17 @@ class LavaMoatPlugin {
                     data: identifierLookup.unenforceableModuleIds || null,
                     json: true,
                   },
+                  {
+                    name: 'ctxm',
```

Comment:
is `ctxm` more esoteric webpack apocrypha?

Reply:
that's short for context modules. it doesn't have to be that ugly. I was in a mood to optimize bundle size prematurely